packages/wallets/src/components/WalletsResetMT5Password/WalletsResetMT5Password.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from 'react';
+import { useTradingPlatformPasswordReset } from '@deriv/api';
+import { PlatformDetails } from '../../features/cfd/constants';
+import useDevice from '../../hooks/useDevice';
+import { TPlatforms } from '../../types';
+import { validPassword } from '../../utils/passwordUtils';
+import { ModalStepWrapper, WalletButton, WalletButtonGroup, WalletPasswordField, WalletText } from '../Base';
+import { WalletPasswordFieldProps } from '../Base/WalletPasswordField/WalletPasswordField';
+import { useModal } from '../ModalProvider';
+import WalletSuccessResetMT5Password from './WalletSuccessResetMT5Password';
+import './WalletsResetMT5Password.scss';
+
+type WalletsResetMT5PasswordProps = {
+    onChange: WalletPasswordFieldProps['onChange'];
+    password: WalletPasswordFieldProps['password'];
+    platform: Exclude<TPlatforms.All, 'ctrader'>;
+    verificationCode: string;
+};
+
+const WalletsResetMT5Password = ({ onChange, password, platform, verificationCode }: WalletsResetMT5PasswordProps) => {
+    const { title } = PlatformDetails[platform];
+    const { mutate, status } = useTradingPlatformPasswordReset();
+    const { hide, show } = useModal();
+    const { isDesktop, isMobile } = useDevice();
+
+    const handleSubmit = () => {
+        mutate({
+            new_password: password,
+            platform,
+            verification_code: verificationCode,
+        });
+    };
+
+    useEffect(() => {
+        if (status === 'success') {
```

Comment:
Why not using IsSuccess, isError, IsLoading instead of checking the status?

Reply:
Apparently `isSuccess` and `isLoading` are not available for `useMutation` in react query.

Reply:
Are you sure? https://tanstack.com/query/v4/docs/react/reference/useMutation

Reply:
I'm not sure if I am doing it wrongly but it seems that it didn't work for me. Aside from that, the whole team is using `status` and is being used allover the application.